code/controllers/subsystem/maptype.dm

```diff
@@ -17,5 +17,5 @@ SUBSYSTEM_DEF(maptype)
 	//The faction tag is unused currently, but the map might break without it
 
 	//All the map tags that delete all jobs and replace them with others.
-	var/list/clearmaps = list()
+	var/list/clearmaps = list("sunnyvale")
```

Comment:
Please don't touch this.
The clear maps variable is to clear ALL jobs and replace with a whole new set.

Reply:
I only copied what I saw in the Reno pr, so shrug.
I'll revert it back to what it was.